casanovo/denovo/model_runner.py

```
@@ -195,11 +201,13 @@ def train(
         self.initialize_data_module(train_paths, valid_paths)
         self.loaders.setup()
 
-        self.trainer.fit(
-            self.model,
-            self.loaders.train_dataloader(),
-            self.loaders.val_dataloader(),
-        )
+        with torch.serialization.safe_globals([numpy.dtype]):
```

Ruff (0.15.0)

[error] Undefined name `numpy`

(F821)